applications/registration-api/src/main/java/no/dcat/controller/CatalogController.java

```diff
@@ -2,7 +2,6 @@
 
 import no.dcat.authorization.EntityNameService;
 import no.dcat.configuration.SpringSecurityContextBean;
-import no.dcat.factory.DatasetFactory;
```

Comment:
Bra at catalog decouples fra datasetfactory. Men begrunnelsen for å gjøre dette, kunne kanskje vært bedre beskrevet i commit message. Vet ikke helt om jeg skjønner hva som menes med at "datacatalog will be published from registration-api, but dataset will be published from search-api."

Reply:
Datakatalog url er lenken til ttl fil, mens dataset url er lenken til dataset
De to har ingenting å gjøre med hverandre blant at Innehold av datakatalog er jo listen av dataset. 
For eksemple, i vårt sak, datakatalog ligger i registration api, mens dataset ligger i search-api.